collector/filesystem_linux.go

```diff
@@ -68,31 +62,35 @@ func (c *filesystemCollector) GetStats() (stats []filesystemStats, err error) {
 		}
 
 		stats = append(stats, filesystemStats{
-			labelValues: labelValues,
-			size:        float64(buf.Blocks) * float64(buf.Bsize),
-			free:        float64(buf.Bfree) * float64(buf.Bsize),
-			avail:       float64(buf.Bavail) * float64(buf.Bsize),
-			files:       float64(buf.Files),
-			filesFree:   float64(buf.Ffree),
-			ro:          ro,
+			labels: filesystemLabels{
+				device:     mpd.device,
```

Comment:
Missed that when changing. Fixed, also renamed 'mpd' to labels to make it more readable.